keras_retinanet/callbacks/eval.py

```diff
@@ -54,20 +61,13 @@ def on_epoch_end(self, epoch, logs=None):
             iou_threshold=self.iou_threshold,
             score_threshold=self.score_threshold,
             max_detections=self.max_detections,
-            save_path=self.save_path
+            save_path=self.save_path,
+            writer=self.tensorboard_image,
+            steps=epoch,
+            number= self.number_images,
         )
-
         # compute per class average precision
-        present_classes = 0
-        precision = 0
-        for label, (average_precision, num_annotations ) in average_precisions.items():
-            if self.verbose == 1:
-                print('{:.0f} instances of class'.format(num_annotations),
-                      self.generator.label_to_name(label), 'with average precision: {:.4f}'.format(average_precision))
-            if num_annotations > 0:
-                present_classes += 1
-                precision       += average_precision
-        self.mean_ap = precision / present_classes
+        self.mean_ap = sum(average_precisions.values()) / len(average_precisions)
```

Comment:
I think this changed at some point in `master`, but your commits seem to undo this?